scripts/create-new-actor.sh

```diff
@@ -125,24 +198,120 @@ cat << EOF
 #include <yaml-cpp/yaml.h>
 
 #include <bsoncxx/json.hpp>
+
 #include <mongocxx/client.hpp>
 #include <mongocxx/collection.hpp>
 #include <mongocxx/database.hpp>
 
 #include <boost/log/trivial.hpp>
+#include <boost/throw_exception.hpp>
 
 #include <gennylib/Cast.hpp>
+#include <gennylib/MongoException.hpp>
 #include <gennylib/context.hpp>
 
 #include <value_generators/DocumentGenerator.hpp>
 
 
 namespace genny::actor {
 
+//
+// The ${q}PhaseLoop<PhaseConfig>${q} type constructs one ${q}PhaseConfig${q} instance
+// for each ${q}Phase:${q} block in your Actor's YAML. We do this at
+// Actor/Workload setup time and before we start recording real metrics.
+// This lets you do any complicated or costly per-Phase setup operations
+// (such as interacting with YAML or doing syntax validations) without
+// impacting your metrics.
+//
+// Imagine you have the following YAML:
+//
+//     SchemaVersion: 2017-07-01
```

Comment:
I feel like a visual representation  workload YAML runs and how it hooks up to the actor would have been useful for me when I first started reading workload files. Dan Pasette expressed a similar sentiment last week in that the phases that are defined next to each other do not run alongside each other IRL.

I'm going use Illustrator to draw a deconstructed diagram when I get back and see if we want to add it to the readme.

Reply:
For sure this can seem a bit "backwards" at first. Genny Ideas doc asks if we should have a "Phase-first syntax" but I think for now just more docs would help. You've inspired me to write up an extended example using multiple instances of HelloWorld with different Repeat/Duration/"background"-ness. A diagram would also be lovely if you manage to put one together!